torch/fx/experimental/proxy_tensor.py

```diff
@@ -1006,7 +1009,10 @@ def __torch_function__(
 
 
 class ProxyTorchDispatchMode(TorchDispatchMode):
-    _managers: List[AbstractContextManager]
+    # Ensure this is read-only; this exists only for legacy reasons
+    @property
+    def enable_tracing(self) -> bool:
+        return True
```

Comment:
Do you have a plan for deleting this? (file an issue if not so we can think about it?)

Reply:
I can delete it in the next PR

Reply:
Actually I can't, it's also blocked on moving the out of tree HOPs in tree